# ESP32 Hub Patch for libraries/BLE/src/BLE2902.h
# Generated: 2024-12-14 10:47:47.112986
# Purpose: Fix BLE memory leaks.
#
--- original/arduino-esp32-3.0.7/libraries/BLE/src/BLE2902.h	2024-12-14 10:40:05
+++ modified/libraries/BLE/src/BLE2902.h	2024-12-13 19:03:11
@@ -23,15 +23,17 @@
  * See also:
  * https://www.bluetooth.com/specifications/gatt/viewer?attributeXmlFile=org.bluetooth.descriptor.gatt.client_characteristic_configuration.xml
  */
-class BLE2902 : public BLEDescriptor {
+class BLE2902 : public BLEDescriptor
+{
 public:
   BLE2902();
+  virtual ~BLE2902() = default;
   bool getNotifications();
   bool getIndications();
   void setNotifications(bool flag);
   void setIndications(bool flag);
 
-};  // BLE2902
+}; // BLE2902
 
 #endif /* CONFIG_BLUEDROID_ENABLED */
 #endif /* SOC_BLE_SUPPORTED */
